LICENSES/Apache-2.0.txt

```diff
@@ -0,0 +1,73 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
```

Comment:
I think we should probably replace the `LICENSE.txt` in root, now that we have separate LICENSES folder with something that refers to these.

Reply:
Unfortunately GitHub is pretty dumb in this regard. According to the [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#applying-a-license-to-a-repository-with-an-existing-license) license picking via UI is only available upon repo creation, _and even then_ all it does is dumps a license file in the repo; it's not real metadata. (I tried it on a dummy repo, just to see.)
![kép](https://github.com/user-attachments/assets/9c0cb630-5678-4eec-bd8e-b562d24aca49)
I guess we have to keep this file for GitHub search's sake and what not.